.github/workflows/ci.yml

```diff
@@ -86,3 +86,20 @@ jobs:
 
       - run: ./build_and_test_wasm32_firefox.sh
       - run: ./build_and_test_wasm32_chrome.sh
+
+  test-arm:
+    name: Test Arm
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - aarch64-unknown-linux-gnu
+          - arm-unknown-linux-gnueabi
+    steps:
+      - uses: actions/checkout@v4
+      - run: rustup update --no-self-update stable
+      - run: rustup default stable
```

Comment:
This is already the default on GitHub images (unsure how far it may lag behind but I've typically gotten `stable` updates on CI within 24h of release), are you sure you need this?

Reply:
I often use the dtolnay action so that I can validate msrv and other things.. but not a huge deal. Definitely important to move off of actions-rs stuff

Reply:
Those help a lot for selecting MSRV yes! `actions-rs` is super deprecated and spamming lots of warnings.

The only advantage their `actions-rs/clippy` currently still has is generating more readable tables from the annotations, IIRC.